WordPress/gutenberg
# [RNMobile] Editor Onboarding: "The Basics" help section -  Topics list UI

**Gutenberg-Mobile PR:** https://github.com/wordpress-mobile/gutenberg-mobile/pull/3702




## NOTE
**This feature is only available in debug builds**


## Description

This PR is the first in a series of PRs to add the Editor onboarding basic help section defined in this ticket: https://github.com/WordPress/gutenberg/issues/30626 and covers the following:
- Adds the "Help" menu option to Gutenberg demo app on Android and iOS to make testing easier. 
- Adds the main help topics list `BottomSheet` and the help topic detail sub bottom sheet. Clicking an item in the help topics list will open the sub-sheet detail page with the appropriate title and back button. The actual details of the help article will be added in a later PR.
- Added additional icons to the `@wordpress/icons` package where needed to match the designs. 

## How has this been tested?



1. Run the demo app
2. Open the menu on iOS by clicking the overflow menu icon in the ActionBar or by shaking the device on Android
3. Tap "Help" - the main help topics bottom sheet should be displayed. 
4. Tap one of the help topics. A new bottom sheet will be displayed. Verify the topic heading matches the topic tapped, and that the `< Back` button is available. 
5. Tap the `< Back` button to go back to the main help topics list. 
6. Tap outside the bottom sheet to dismiss. 



https://user-images.githubusercontent.com/5810477/124996585-2e43d880-e017-11eb-8fd7-cc4ae52038de.mp4



## Screenshots 

Light | Dark
-- | --
![help-topics-android](https://user-images.githubusercontent.com/5810477/124687363-bc4a8280-dea2-11eb-96bf-5dafb6eb35d4.png)|![help-topics-dark-android](https://user-images.githubusercontent.com/5810477/124687368-beacdc80-dea2-11eb-869c-c9599f9bed57.png)
![help-topics-ios](https://user-images.githubusercontent.com/5810477/124687374-c40a2700-dea2-11eb-884d-766e1e0fb255.png)|![help-topics-dark-ios](https://user-images.githubusercontent.com/5810477/124687378-c5d3ea80-dea2-11eb-9e44-e897bbf2eb6e.png)


## Types of changes




New feature. Only available in non-production builds. 

## Checklist:
